Microsoft.Azure.Cosmos/src/Patch/PatchSpecification.cs

```diff
@@ -0,0 +1,118 @@
+﻿//------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//------------------------------------------------------------
+
+namespace Microsoft.Azure.Cosmos.Patch
```

Comment:
Please keep in the default namespace. This caused a lot of negative feedback in v2 SDK.
```suggestion
namespace Microsoft.Azure.Cosmos
```